### PR TITLE
OKTA-423084 OIE clean up self-registration for Java

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Rework the embedded SDK self-registration use case for Java (match the steps with the diagram). Fix the common "Configuration updates" for the self-registration use case to match new common "Get set up" guide. 
- **Is this PR related to a Monolith release?** No
### Resolves:

* [OKTA-423084](https://oktainc.atlassian.net/browse/OKTA-423084)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/integrationsteps.md [hunk 1]
```diff
@@ -114,15 +114,15 @@ VerifyAuthenticatorOptions verifyAuthenticatorOptions = new VerifyAuthenticatorO
 AuthenticationResponse authenticationResponse = idxAuthenticationWrapper.verifyAuthenticator(proceedContext, verifyAuthenticatorOptions);
 ```
 
-The request returns an `AuthenticationResponse` object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` property and an `Authenticators` list containing the email and phone factors. Reuse the authenticator enrollment form from [Step 3: Display enrollment factors](#step-3-display-enrollment-factors) to display the list of authenticators to the user.
+The request returns an `AuthenticationResponse` object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` property and an `Authenticators` list containing the email and phone factors. Reuse the authenticator enrollment form from step [3: Display the enrollment authenticators](#_3-display-the-enrollment-authenticators) to display the list of authenticators to the user.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/integrationsteps.md:117]:
```suggestion
The request returns an `AuthenticationResponse` object with the `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` property and an `Authenticators` list containing the email and phone factors. Reuse the authenticator enrollment form from step [3: Display the enrollment authenticators](#_3-display-the-enrollment-authenticators) to display the list of authenticators to the user.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/integrationsteps.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ AuthenticationResponse authenticationResponse =
    idxAuthenticationWrapper.verifyAuthenticator(proceedContext, verifyAuthenticatorOptions);
 ```
 
-If the request to verify the code is successful, the SDK returns an `AuthenticationResponse` object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` property and an `Authenticators` list containing the phone factor. Reuse the authenticator enrollment form from [Step 3: Display enrollment factors](#step-3-display-enrollment-factors) to display the list of authenticators to the user.
+If the request to verify the code is successful, the SDK returns an `AuthenticationResponse` object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` property and an `Authenticators` list containing the phone factor. Reuse the authenticator enrollment form from step [3: Display the enrollment authenticators](#_3-display-the-enrollment-authenticators) to display the list of authenticators to the user.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/integrationsteps.md:153]:
```suggestion
If the request to verify the code is successful, the SDK returns an `AuthenticationResponse` object with the `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` property and an `Authenticators` list containing the phone factor. Reuse the authenticator enrollment form from step [3: Display the enrollment authenticators](#_3-display-the-enrollment-authenticators) to display the list of authenticators to the user.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/summaryofsteps.md [hunk 3]
```diff
@@ -1,45 +1,45 @@
 The sequence of steps for self-service registration is described in the following three sequence diagrams:
 
-### Start new user registration with password factor
+### Start a new user registration with the password authenticator
 
-The following diagram illustrates the beginning of the registration process where the user initiates their sign-in and enters their password.
+The following diagram illustrates the beginning of the registration process where the user initiates the sign-in flow and enters their password.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/summaryofsteps.md:5]:
```suggestion
The following diagram illustrates the beginning of the registration process when the user initiates the sign-in flow and enters their password.
```

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/summaryofsteps.md:5]: I'm pretty sure it should be "when", but I confess I'm not 100% sure.

**Review by brentschaus-okta — APPROVED** [review]
Try to approve again. I'm not appearing on the devdocs team for this branch?

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/summaryofsteps.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/summaryofsteps.md:5]: when does sound better :-)

**Pull request merged**